src/modules/APIdata.js

```diff
@@ -0,0 +1,48 @@
+export class DataAPI {
+  constructor() {
+    this.url = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/NIsz8fkt26lmn27CsIOPQ/scores/';
+    this.scoreList = document.querySelector('.scores-content');
+  }
+
+  getData = async () => {
+    const response = await fetch(this.url);
+    const data = await response.json();
+    this.displayData(data.result);
+  }
+
+  setData = async (player, newScore) => {
+    await fetch(this.url, {
+      method: 'POST',
+      body: JSON.stringify({
+        user: player,
+        score: newScore,
+      }),
+      headers: {
+        'Content-type': 'application/json; charset=UTF-8',
+      },
+    })
+      .then((response) => response.json())
+      .then((data) => {
+        console.log(data);
```

Comment:
- [ ] For best JavaScript practices kindly try to not commit ```console.log``` to your repo, It’s ugly, it kills performance and it can make confidential data be visible to anyone using the browser tools to look at your website.